how-to/how-to-navigate-local-logs.md

```diff
@@ -0,0 +1,32 @@
+---
+title: How to Navigate to The Extension for Visual Studio and The Classic Windows Desktop Client Logs
+description: Step by step tutorial for navigating to the Telerik Platform local Windows logs.
+type: how-to
+page_title: How to Navigate to AppBuilder Windows Clients Local Logs
+slug: navigate-local-logs
+position:
+tags: other
+teampulseid: 
+ticketid: 
+pitsid: 
+---
+                    
+## Environment
+<table>
+  <tr>
+    <td>{{site.ab-s}} ({{site.ab}}) clients</td>
+    <td>Extension for Visual Studio, classic Windows Desktop</td>	
+  </tr>
+</table>
+
+## Description
+
+This article provides the general steps for navigating to the Extension for Visual Studio and the classic Windows Desktop client logs.
```

Comment:
You can add one sentence about what is the use of these logs—that AppBuilder stores crash logs and you can use them to diagnose problems or something of this sort.